python/pdstools/adm/ADMDatamart.py

```diff
@@ -24,6 +22,9 @@
 from .ADMTrees import ADMTrees
 from .Tables import Tables
 
+logger = logging.getLogger("PDSTools.ADMDatamart")
```

Comment:
I would simply do `logging.getLogger(__name__)`. No need to customize - `__name__` would already produce a unique path

Reply:
I am using a child of the logger created in Streamlit app which has a log_buffer. If I initialize the buffer with ```__name__``` in ADMDatamart I can't combine the logs from the app and ADMDatamart.